staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_discovery_controller.go

```diff
@@ -96,7 +96,21 @@ func (c *DiscoveryController) sync(version schema.GroupVersion) error {
 			Version:      crd.Spec.Version,
 		})
 
-		if crd.Spec.Version != version.Version {
+		foundThisVersion := false
+		for _, v := range crd.Spec.Versions {
```

Comment:
if we're iterating over and adding all served versions, we can drop the append on line 94 of spec.Version, right?

Reply:
+1